stdlib/public/Distributed/DistributedAssertions.swift

```diff
@@ -99,25 +120,39 @@ extension DistributedActor {
-  /// serial executor, or crash.
+  /// Assume that the current task is executing on this (local) distributed
+  /// actor's serial executor, or stop program execution otherwise.
+  ///
+  /// This method allows to *assume and verify* that the currently
+  /// executing synchronous function is actually executing on the serial
+  /// executor of the this (local) distributed actor.
   ///
-  /// This method allows developers to *assume and verify* that the currently executing synchronous function
-  /// is actually executing on the serial executor that this distributed (local) actor is using.
+  /// If that is the case, the operation is invoked isolated to the main actor
+  /// (`@MainActor () -> T`), allowing synchronous access to actor local state
+  /// without hopping through asynchronous boundaries.
   ///
-  /// If that is the case, the operation is invoked with an `isolated` version of the actoe,
-  /// allowing synchronous access to actor local state without hopping through asynchronous boundaries.
+  /// If the current context is not running on the actor's serial executor,
+  /// this method will crash with a fatal error (similar
+  /// to ``preconditionIsolated()``).
   ///
-  /// If the current context is not running on the actor's serial executor, or if the actor is a reference to a remote actor,
-  /// this method will crash with a fatalError (similar to ``preconditionIsolated()``).
+  /// This method can only be used from synchronous functions, as asynchronous
+  /// functions should instead perform a normal method call to the actor, which
+  /// will hop task execution to the target actor if necessary.
   ///
-  /// This method can only be used from synchronous functions, as asynchronous ones should instead
-  /// perform normal method call to the actor.
+  /// - Note: This check is performed against the actor's serial executor,
+  ///   meaning that / if another actor uses the same serial executor--by using
+  ///   another actor's executor as its own ``DistributedActor/unownedExecutor``
+  ///   --this check will succeed , as from a concurrency safety perspective,
+  ///   the serial executor guarantees mutual exclusion of those two actors.
   ///
   /// - Parameters:
-  ///   - operation: the operation that will be executed if the current context is executing on the actors serial executor, and the actor is a local reference
-  ///   - file: source location where the assume call is made
-  ///   - file: source location where the assume call is made
+  ///   - operation: the operation that will be executed if the current context
+  ///                is executing on the actors serial executor, and the actor
+  ///                is a local reference.
+  ///   - file: The file name to print if the assertion fails. The default is
+  ///           where this method was called.
+  ///   - line: The line number to print if the assertion fails The default is
+  ///           where this method was called.
```

Comment:
By the way, you don't have to indent to match the `:` like this.  It's often easier to indent the 2nd and later lines by two spaces, so they line up under the `- ` that starts the parameter.  But if you prefer this way and don't mind the extra work, this is ok too.  DocC supports both as far as I know.

Reply:
okey will do :) the alignment uses up much space I guess. Visually i like it but since we're trying to fit in 80chars width this gets too wide I guess. Thanks